js/chat/chat.ts

```diff
@@ -76,6 +76,13 @@ class ChatMessage extends LightElement {
 
   updated(changedProperties: Map<string, unknown>): void {
     if (changedProperties.has("content")) {
+      this.dispatchEvent(
+        new CustomEvent("shiny-chat-message-content-updated", {
+          bubbles: true,
+          composed: true,
+        })
+      );
```

Comment:
Note that this event will fire when either a (full) message is appended (i.e., `append_message()`), or if just the `content` attribute of a message is updated (i.e., `append_message_stream()`)